crates/uv-python/src/discovery.rs

```diff
@@ -1477,6 +1479,25 @@ impl VersionRequest {
                 Some(python),
                 None,
             ],
+            Self::FreeThreaded(version) => version
```

Comment:
e.g., if you didn't use a recursive enum we could just generate the names properly in `VersionRequest::default_names` instead of mutating the names here.